src/logging.c

```diff
@@ -367,17 +366,27 @@ void log_print(int lvl, const char* format, ...) {
  * Log the message with the given level. New line character
  *              is appended to the error stream.
  * @param lvl     level of the message
+ * @param file    filename where the log occurred
+ * @param line    line number in the file
+ * @param lvl     level of the message
  * @param format  format of the message
  *            ... - the additional arguments
  */
 
-void log_println(int lvl, const char* format, ...) {
+void log_println_impl(int lvl, const char* file, int line, const char* format, ...) {
```

Comment:
I agree on both. I also don't know what value is provided by I2ErrLogVT, but tearing it out would be a lot of work and would likely break a few things in small ways.  IMO a long term goal of NDT should be to get rid of the I2Util libraries entirely.